node/src/keyshare/local.rs

```diff
@@ -22,7 +23,7 @@ impl LocalKeyshareStorage {
 
 #[async_trait::async_trait]
 impl KeyshareStorage for LocalKeyshareStorage {
-    async fn load(&self) -> anyhow::Result<Option<RootKeyshareData>> {
+    async fn load(&self) -> anyhow::Result<Option<PartialRootKeyshareData>> {
```

Comment:
unrelated to this PR, but I don't see why this function needs to be async

Reply:
oh, that's why there was this commit: https://github.com/Near-One/mpc/pull/267/commits/14477ea299011a62f3362c9d8d2e7bb8a4641efd that I've reverted...
Now it makes sense